src/test/scala/idfs-tests.scala

```diff
@@ -10,8 +10,8 @@ final class IdfsTests {
   @Test
   def mountHomeDir(): Unit = {
     val fs  = idfs(homeDir) mount mnt
-    val s1 = mnt.ls.map(_.filename).sorted.mkString(" ")
-    val s2 = homeDir.ls.map(_.filename).sorted.mkString(" ")
+    val s1 = mnt.ls.map(_.name).sorted.mkString(" ")
+    val s2 = homeDir.ls.map(_.name).sorted.mkString(" ")
```

Comment:
I almost always prefer the shorter name, but I have had to pay too many mental taxes on members called name. I prefer filename here, or pathname, or simpleName.